src/tabs/tab.jsx

```diff
@@ -118,6 +125,22 @@ const Tab = React.createClass({
       width: width,
     }, style);
 
+    let iconElement;
+    if (icon) {
+      let params = {
+        style: {
+          fontSize: 24,
+          display: label ? 'block' : 'inline-block',
+          color: selected ? this.state.muiTheme.tabs.selectedTextColor : this.state.muiTheme.tabs.textColor,
+        },
+      };
+      // If it's svg icon set color via props
+      if (icon.type.displayName !== 'FontIcon') {
+        params.color = selected ? this.state.muiTheme.tabs.selectedTextColor : this.state.muiTheme.tabs.textColor;
```

Comment:
It's for `SvgIcon` they need a color props not a style color else color will not change.